.github/CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# More details are here: https://help.github.com/articles/about-codeowners/
+
+# The '*' pattern is global owners.
+
+# Order is important. The last matching pattern has the most precedence.
+# The folders are ordered as follows:
+
+# In each subsection folders are ordered first by depth, then alphabetically.
+# This should make it easy to add new rules without breaking existing ones.
+
+# Documentation owner: TBD
```

Comment:
I volunteer if no one else wants to own this.

Reply:
Sure. I thought you already were unofficially, or were going to be.

Reply:
Totally, just making it clear for anyone else who wasn't privy to our earlier conversation(s) on the subject. :smile: